src/client/components/DisasterQuestion/index.js

```diff
@@ -54,18 +54,18 @@ function DisasterQuestion(props) {
     >
       <fieldset>
         <Form.Label as="legend">{props.questionText}</Form.Label>
-        {disasterRadioChoices.map((value) => (
+        {disasterRadioChoices.map((value, index) => (
           <Form.Check
             key={value}
             label={value}
             type="radio"
             id={`disaster-question-radio${value}`}
             onChange={onChange}
             name="disaster-question"
-            value={value}
+            value={`choice-${index + 1}`}
```

Comment:
This means we just save choice-5 or whatever in the DB. I think we need to add logic to save the actual answer in the DB.

Reply:
Oh, we need to save the actual text in the db? Ugh, ok.

Reply:
I think we should save something more identifiable than just the number of the choice. What if we re-order the choices for some reason? I don't think it needs to be the exact text.

Reply:
Ok, answer updated to be "choice-5 [answer text ....]".

Reply:
now that I'm making the detailed confirmation page, I feel like there must've been a cleaner solution to this :P if we think the order of the choices might change, why store the key-with-order in the database at all? I'll just drop everything before the first whitespace when retrieving, I guess

Reply:
If the wording changes, then we need to store the key! I guess a better way would have been to rename the string keys to use a short name rather than a number and store that. Sorry, I didn't think of it at the time!

Yeah, it should be safe to just drop everything before the first whitespace. Sorry for the hackish code. :/